embrace_mobile/metadata.csv

```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```

Comment:
I believe we can remove this since this App doesn't contain an `integration` section in the manifest
(which should get past the current CI errors)